### PR TITLE
Move scimitar from Medieval Stuff mod

### PR DESCRIPTION
**Pull request by Rail-Runner**
Move scimitar and its' fake version from Medieval Stuff mod into the main game. Also had to move a book as it the recipe required to craft scimitar is learnable only by reading it.

Its' damage per 100 moves is 31.3.

Other changes:
1. Reduce spawn rate of scimitar in some item groups.
2. Add inferior scimitar to some item groups.
3. Add China arms book. scimitar and its' fake version to No Medieval Items mod's blacklist.
4. Add some recipes to China arms book, so that it wouldn't contain just a single recipe for now.

### CODE REVIEW

**Comment by chaosvolt** [comment]
Ah, interesting. I was worried that the recipe book would cause issue once one of the items made by it becomes up for mainlining.

Also of note, Arms and Armor of China includes a mixture of weapons, armor, and also black gunpowder. Which is perfectly fine for a mod book, but I was further worried it would be seen as inconsistent with the mainline weapon and armor books being divided up.

File: data/json/items/melee.json [hunk 1]
```diff
@@ -3805,6 +3805,31 @@
     "use_action": { "type": "knife" },
     "category" : "weapons",
     "qualities": [["CUT", 1], ["BUTCHER", 8]]
+    },{
+    "id": "scimitar",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "scimitar",
+    "description": "This is a curved sword associated with various middle eastern and central asian countries. Designed for slashing, and quite deadly against unarmored targets.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: Please capitalize regions: "Middle Eastern and Central Asian countries" and put double space after a period. (It wasn't fixed in initial implementation but we have a second chance).

File: data/json/items/melee.json [hunk 2]
```diff
@@ -3832,6 +3857,23 @@
     "category" : "weapons",
     "qualities": [["CUT", 1], ["BUTCHER", 8]]
   },{
+    "id": "scimitar_fake",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "scimitar",
+    "description": "This is a dull, cheaply made replica of a curved sword, associated with various middle eastern and central asian countries.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: The same stuff about capitalization here.

File: data/json/items/books.json [hunk 3]
```diff
@@ -3060,6 +3060,25 @@
...
         "required_level" : 4
+    },{
+        "type" : "BOOK",
+        "id" : "textbook_armschina",
+        "name" : "Arms and Armor of Imperial China",
+        "name_plural" : "Arms and Armor of Imperial China",
+        "max_level" : 10,
+        "description" : "This textbook details the military history of ancient China, with a particular focus on the nature of historical arms and armor.  It also details some of the equipment used by other cultures that came into conflict with the empire over various dynasties.",
+        "weight" : 654,
+        "to_hit" : -2,
```
Comment (illi-kun) [inline data/json/items/books.json]: Huh, you updated this parameter to correct value, nice to see it!

File: data/json/items/melee.json (hunk 1 above)
Comment (chaosvolt) [inline data/json/items/melee.json]: Ack. I should've remembered to do that when it was mentioned in the big mainlining PR. @_@

**Pull request merged**